handlers.go

```diff
@@ -84,6 +86,16 @@ func (dht *IpfsDHT) handleGetValue(ctx context.Context, p peer.ID, pmes *pb.Mess
 		}
 
 		resp.CloserPeers = pb.PeerInfosToPBPeers(dht.host.Network(), closerinfos)
+
```

Comment:
What's the reason to send all unsigned addresses from our peerstore instead of just taking the signed addresses and extracting them?